core_components/bot_core/component_management.go

```diff
@@ -19,7 +19,7 @@
 package bot_core
 
 // registerAvailableComponents ensures that all available components
-// are registered in the database.
+// are registered in the entities.
```

Comment:
Refactor Issue?

Reply:
This method is dead code an has been dropped.